packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -263,7 +264,7 @@ class TransactionService {
           const pageLoadTransactionName = this._config.get(
             'pageLoadTransactionName'
           )
-          if (name === NAME_UNKNOWN && pageLoadTransactionName) {
```

Comment:
history pushState test fails due to this change. Do we want this logic to exist here @hamid?